apps/backend/src/app/api/admin/users/[id]/route.ts

```diff
@@ -65,6 +66,33 @@ class UserRouteWrapper {
       )
     }
   }
+
+  /**
+   * DELETE method to delete a single user by ID.
+   *
+   * @param req The request object
+   * @param params The route parameters containing the user ID
```

Comment:
this should be _req and not req as it is not being used

```suggestion
   *
   * @param _req The request object
   * @param params The route parameters containing the user ID
```